tools/tool-env/isopackets.js

```diff
@@ -227,8 +227,10 @@ var newIsopacketBuildingCatalog = function () {
       // expect them to all build, and we're fine with dying if they don't
       // (there's no worries about needing to springboard).
       isopacketCatalog.initialize({
-        localPackageSearchDirs: [files.pathJoin(
-          files.getCurrentToolsDir(), 'packages')],
+        localPackageSearchDirs: [
+          files.pathJoin(files.getCurrentToolsDir(), 'packages'),
+          files.pathJoin(files.getCurrentToolsDir(), 'packages-for-isopackets', 'blaze', 'packages')
```

Comment:
Nice! If/when we turn the command-line tool into an npm package, we can drop the `packages` directory and move everything isopacket-related into `packages-for-isopackets`.

Last I checked, that list was relatively short:
```
babel-compiler
babel-runtime
base64
blaze
blaze-tools
boilerplate-generator
check
constraint-solver
contsraint-solver
ddp-client
ddp-common
ddp-rate-limiter
ddp-server
deps
diff-sequence
ecmascript
ecmascript-runtime
ejson
html-tools
htmljs
id-map
jquery
logging
logic-solver
meteor
modules
modules-runtime
mongo-id
npm-mongo
observe-sequence
package-version-parser
promise
random
reactive-var
retry
spacebars
spacebars-compiler
tracker
ui
underscore
webapp-hashing
xmlbuilder
```